src/components/items/item.js

```diff
@@ -28,6 +28,7 @@ const Item = ({
   >
     {!group && (
       <Checkbox
+        id={`${item.id || classnames(styles.item)}`}
```

Comment:
hi @BaptWaels, I think this is a little bit too specific for shared component: we do support items list without `id` field. For that case component will have none-unique id as classnames(styles.item) generates the same value for all checkboxes.
Also, it may be better to not just pass input item.id as it is but to have a prefix. 
Something like 
```
<Checkbox
    id={item.id ? `checkbox-${item.id}` : null}
```
Thanks

Reply:
Good idea, thank you for the feedback. I've just updated my PR